.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "overrides": [
+    {
+      "files": ["**/*.cy.js"],
```

Comment:
If you are getting lint errors on `cypress.config.json` you can also add it to `files` here